google/location.go

```diff
@@ -113,22 +113,34 @@ func (l *Location) RemoveContainer(id string) error {
 // ItemByURL retrieves a stow.Item by parsing the URL, in this
 // case an item is an object.
 func (l *Location) ItemByURL(url *url.URL) (stow.Item, error) {
```

Comment:
Adapting this function was required for `google/stow_test.go` to pass.

Note that on master and on this branch I needed to disable [this](https://github.com/flyteorg/stow/blob/632adef717d32cc01727f9ec5b064c0505bbffb9/test/test.go#L66) check for the test to pass: `t.Fatalf("Leaked %d TCP file descriptor", diffFDs)`.

Reply:
interesting. Can you paste the error? I'm unable to repro it on a Mac (I also added https://github.com/flyteorg/stow/pull/18 and all tests pass on linux/amd64 too).